lib/pbench/test/unit/server/conftest.py

```diff
@@ -259,23 +262,33 @@ def fake_stat(file: str):
 
 
 @pytest.fixture()
-def attach_dataset(pbench_token, create_user):
+def attach_dataset(create_drb_user, create_user):
     """
     Create test Datasets for the authorized user ("drb") and for another user
     ("test")
 
     Args:
-        pbench_token: create a "drb" user and authorize it
+        create_drb_user: create a "drb" user
         create_user: create a "test" user
     """
-    Dataset(
-        owner="drb",
-        controller="node",
-        name="drb",
-        access="private",
-        md5="random_md5_string1",
-    ).add()
-    Dataset(owner="test", controller="node", name="test", access="private").add()
+    with freeze_time("1970-01-01 00:42:00"):
+        Dataset(
+            owner="drb",
+            created=datetime.datetime(2020, 2, 15),
+            uploaded=datetime.datetime(2022, 1, 1),
+            controller="node",
+            name="drb",
+            access="private",
+            md5="random_md5_string1",
+        ).add()
+        Dataset(
+            owner="test",
+            created=datetime.datetime(2002, 5, 16),
+            controller="node",
+            name="test",
+            access="private",
+            md5="random_md5_string2",
+        ).add()
```

Comment:
Yes; it should be defaulted to the current (frozen) time, and that seemed worth showing; that's the normal and expected mode. In fact, specifying `uploaded` should be extremely unusual; but as it's possible, I decided to do that in one of the canned datasets "just because".

Reply:
Sounds good.  It is probably worth a comment to that effect.